charged-up-2023/software/intake-control.md

```diff
@@ -0,0 +1,2 @@
+# Programming the Intake 
```

Comment:
I see you don't have this linked anywhere yet. Are you planning on linking it after it's more complete?

Reply:
Yep! These are just some intital changes I wanted to make, more does need to be updated asap